modules/network-observability-cli-capturing-packets.adoc

```diff
@@ -24,6 +24,7 @@ $ oc netobserv packets --filter=tcp,80
 ----
```

Comment:
L19 above needs to be updated too. As suggested in comment https://github.com/openshift/openshift-docs/pull/81064#discussion_r1759260706

Reply:
Ah good catch here. I updated to the same as the example in the Packet capture options table in the reference.

Reply:
`oc netobserv packets --action=Accept --cidr=0.0.0.0/0 --protocol=TCP --port=49051`